src/pages/common/Error.tsx

```
@@ -1,3 +1,20 @@
+import Button from '@/components/common/Button';
+
+import AlertCircle from '@/assets/icons/alert-circle_Fill.svg?react';
+
 export default function Error() {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)